lib/esty_commist.rb

```diff
@@ -0,0 +1,26 @@
+require 'httparty'
```

Comment:
Do we need this file in lib or is this leftover from just starting the logic and things got moved around?

Reply:
Did not need any of that. I have deleted commist file